packages/host/tests/integration/store-test.gts

```diff
@@ -533,6 +533,63 @@ module('Integration | Store', function (hooks) {
     (instance as any).name = 'Paper';
   });
 
+  test<TestContextWithSave>('an unsaved instance will auto save when its data changes', async function (assert) {
+    assert.expect(2);
+    let instance = new PersonDef({ name: 'Andrea' });
+    await store.add(instance, { doNotPersist: true });
+
+    this.onSave((url, doc) => {
+      assert.strictEqual(
+        url.href.split('/').pop()!,
+        instance[localId],
+        'the new card url ends with the local id',
+      );
+      assert.strictEqual(
+        (doc as SingleCardDocument).data.attributes?.name,
+        'Air',
+        'card data is correct',
+      );
+    });
+
+    (instance as any).name = 'Air';
+  });
+
+  test<TestContextWithSave>('getSaveState works for initially unsaved instance', async function (assert) {
+    let instance = new PersonDef({ name: 'Andrea' });
+    await store.add(instance, { doNotPersist: true });
+
+    assert.strictEqual(
+      store.getSaveState(instance[localId]),
+      undefined,
+      'save state is undefined',
+    );
+
+    (instance as any).name = 'Air';
+
+    assert.strictEqual(
+      store.getSaveState(instance[localId])?.isSaving,
+      true,
+      'isSaving state is correct',
+    );
+
+    await waitUntil(() => store.getSaveState(instance[localId])?.lastSaved);
+
+    assert.strictEqual(
+      store.getSaveState(instance[localId])?.isSaving,
+      false,
+      'isSaving state is correct',
+    );
+    assert.strictEqual(
+      store.getSaveState(instance.id)?.isSaving,
+      false,
+      'isSaving state is correct (by remote id)',
+    );
```

Comment:
Does `assert.false` work for these? And `assert.true` above

Reply:
sure no prob